plugins/azure/defender/enableDefenderForPostgresql.js

```diff
@@ -0,0 +1,43 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Enable Defender For PostgreSQL Flexible Servers',
+    category: 'Defender',
+    domain: 'Management and Governance',
+    description: 'Ensures that Microsoft Defender is enabled for Azure PostgreSQL Flexible Servers.',
+    more_info: 'Enabling Defender for Cloud on PostgreSQL Flexible Servers allows detection of unusual database access, query patterns, and suspicious activities, enhancing overall security.',
+    recommended_action: 'Enable Microsoft Defender for PostgreSQL Flexible Servers in Defender plans for the subscription.',
+    link: 'https://learn.microsoft.com/en-us/azure/postgresql/flexible-server/concepts-security#microsoft-defender-for-cloud-support',
+    apis: ['pricings:list'],
+
+    run: function(cache, settings, callback) {
+        var results = [];
+        var source = {};
+        var locations = helpers.locations(settings.govcloud);
+
+        async.each(locations.pricings, function(location, rcb) {
+            var pricings = helpers.addSource(cache, source,
+                ['pricings', 'list', location]);
+
+            if (!pricings) return rcb();
+
+            if (pricings.err || !pricings.data) {
+                helpers.addResult(results, 3,
+                    'Unable to query for Pricing: ' + helpers.addError(pricings), location);
+                return rcb();
+            }
+
+            if (!pricings.data.length) {
+                helpers.addResult(results, 0, 'No Pricing information found', location);
+                return rcb();
+            }
+
+            helpers.checkMicrosoftDefender(pricings, 'opensourcerelationaldatabases', 'PostgreSQL Flexible Servers', results, location);
```

Comment:
```suggestion
            helpers.checkMicrosoftDefender(pricings, 'opensourcerelationaldatabases', 'PostgreSQL Flexible Servers', results, location);
```
this plan is for multiple types of databases, we should remove PostgreSQL Flexible Server from the title and make this plugin generic for open source relational databases